schema-engine/connectors/schema-connector/src/warnings.rs

```diff
@@ -134,19 +276,19 @@ impl fmt::Display for Warnings {
             f,
         )?;
 
-        render_warnings(
+        render_warnings_grouped(
```

Comment:
It's not immediately clear to me how the scaffolding for `render_warnings_grouped` works. I.e. the workflow for adding new warnings - or editing previous warnings - that we want grouped. The change that @tomhoule called out does seem a lot simpler to me